### DISCUSSION TITLE
The program closes unexpectedly during rendering

### DISCUSSION THREAD

**Discussion by ghost** [ntsc-rs/ntsc-rs#99] [Q&A]
I have problems with the program. During rendering, the program closes while rendering. I tried several times and the same thing closes during rendering.

I have attached a video demonstrating the problem.

https://mega.nz/file/VGNjEARY#6l_cLOJziz1Um724tS9pu2in45RhP1pIaTdMUcJ5G-I

**Comment by valadaptive** [comment]
Not sure what could be causing this. You're not out of disk space, are you?

You could also try changing the codec to FFV1--not sure if it's something specific with the H.264 codec.

**Comment by ghost** [comment]
No, I have more space on my HDD

**Reply by Degamisu** [reply]
> No, I have more space on my HDD

is this your main? (the one with your OS installed on)

**Reply by ghost** [reply]
Win11 Pro 23H2

**Reply by Degamisu** [reply]
no is the HDD the one with windows on it?

**Reply by ghost** [reply]
Yes

**Comment by ghost** [comment]
No, it didn't help

**Comment by valadaptive** [comment]
If you run it from the command prompt, are there any errors when it closes?

**Comment by ghost** [comment]
No, the same thing remained

**Comment by valadaptive** [comment]
Were there any error messages in the command prompt? Can you send a screenshot of the command prompt window after the crash?

**Comment by ghost** [comment]
![After failure](https://github.com/valadaptive/ntsc-rs/assets/115736491/0ff754ac-a4a6-4443-9d51-bfc10ee59e00)
![Before failure](https://github.com/valadaptive/ntsc-rs/assets/115736491/1ed9a220-2615-4353-a6e7-364bf73dc98b)
![Rendering process](https://github.com/valadaptive/ntsc-rs/assets/115736491/e176e774-f83a-4678-9db9-437229e88245)
I note that the command line did not display any errors or other problems related to this program

**Reply by valadaptive** [reply]
Can you try it again with version 0.7.0? The fix in 0.7.1 for disabling the command prompt showing up has the side effect of hiding all potential error messages.

**Reply by ghost** [reply]
No. Nothing! Without changes

cmd output

Microsoft Windows [Version 10.0.22631.3593]
(c) Корпорация Майкрософт (Microsoft Corporation). Все права защищены.

C:\Windows\System32>cd C:\Users\Илья\Downloads\ntsc-rs-windows-standalone (1)\ntsc-rs-windows-standalone\bin

C:\Users\Илья\Downloads\ntsc-rs-windows-standalone (1)\ntsc-rs-windows-standalone\bin>ntsc-rs-standalone.exe

C:\Users\Илья\Downloads\ntsc-rs-windows-standalone (1)\ntsc-rs-windows-standalone\bin>

**Reply by valadaptive** [reply]
Not entirely sure what could be causing this then. What kind of CPU do you have?

**Reply by ghost** [reply]
AMD Ryzen 5 2600

**Reply by valadaptive** [reply]
Can you ensure that PBO (overclocking) is explicitly set to "Disabled" in your BIOS and try again? A lot of motherboards enable it by default.

**Reply by ghost** [reply]
I not enable PBO in BIOS
